contrib/debian/control.in

```diff
@@ -198,3 +198,9 @@ Depends: ${shlibs:Depends}, ${misc:Depends}, make | build-essential | dpkg-dev
 Description: Template for signed fwupd package
  This package is used to control code signing by the Debian signing
  service.
+
+Package: fwupd-qubes-vm-whonix
```

Comment:
I'm leaning on we should have some sort of flag that determines whether this binary package gets built as part of https://github.com/fwupd/fwupd/blob/master/contrib/ci/generate_debian.py

I don't think it should be done in general purpose Debian or Ubuntu, right?  It should only be when the Debian derivatives Whonix generates the binary packages.

Reply:
> I don't think it should be done in general purpose Debian or Ubuntu, right?

That's right. It's Qubes specific package. 

> I'm leaning on we should have some sort of flag that determines whether this binary package gets built as part of https://github.com/fwupd/fwupd/blob/master/contrib/ci/generate_debian.py

It could be handled as another OS environment variable e.g. `Debian-x86_64-Whonix` and it would be added to the `TARGET` split: https://github.com/fwupd/fwupd/blob/master/contrib/ci/generate_debian.py#L24

Reply:
Sounds like a good enough approach to me.  If you find while you're doing it that doesn't end up scaling well enough feel free to add `argparse` support there instead though.  The environment parsing was just a convenient way for our first CI jobs to do it.

Reply:
There is a possibility to generate separate `fedora-qubes` and `debian-x86_64-qubes` containers that build qubes packages. Standard containers don't provide qubes DEBs and RPMs.